CHANGELOG.rst

```diff
@@ -2,6 +2,215 @@
 Release Notes
 =============
 
+6.0.0-rc.1
+==========
+
+Prelude
+-------
+
+The execution of the main agent, trace-agent (APM), and process-agent processes is now orchestrated
```

Comment:
I sometimes use the prelude, but I see no one else is doing it, so we don't want to make it look like these three changes are maybe more important than others. Feel free to kill the prelude, or not.

Reply:
I'd say the orchestration and the changes to where the trace and process agent should be configured in are pretty significant changes, I don't mind leaving them here.

But you're right, we may want to remove the prelude entirely from the individual release notes (and maybe manually write "highlights" for every release to stress on the most significant changes for users?)